editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-component-renderer.tsx

```diff
@@ -34,7 +36,14 @@ export function isComponentRendererComponent(
 export function createComponentRendererComponent(params: {
   topLevelElementName: string
 }): ComponentRendererComponent {
-  const Component = (realPassedProps: any) => {
+  const Component = (realPassedPropsIncludingUtopiaSpecialStuff: any) => {
+    const {
+      [UTOPIA_SCENE_PATH]: scenePathAny, // TODO types?
+      ...realPassedProps
+    } = realPassedPropsIncludingUtopiaSpecialStuff
+
+    const scenePath: ScenePath | null = TP.isScenePath(scenePathAny) ? scenePathAny : null
```

Comment:
We still appear to have a case where the scene path prop isn't set, which is captured in the test [here](https://github.com/concrete-utopia/utopia/blob/master/editor/src/components/canvas/ui-jsx-canvas-bugs.spec.tsx). To get around that we support null template paths now where we didn't before, but it's still a bit confusing whether or not that is actually a legitimate case or a bug elsewhere in the logic... (the commit for supporting those null template paths is https://github.com/concrete-utopia/utopia/pull/972/commits/afab4a59099262bf059d0dfcede95cd7658ed6c8)